### PR TITLE
Adds an are you sure dialog when pressing delete note.

### PR DESCRIPTION
**Pull request by DaxDupont**
[Changelogs]: 

:cl: Dax Dupont
admin: Removing notes now has an "Are you sure" dialog.
/:cl:

[why]: Because I am a fucking idiot and I accidentally double clicked when clicking the view older notes link and hit the delete button that was under it.

### CODE REVIEW

File: code/modules/admin/topic.dm [hunk 1]
```diff
@@ -1092,14 +1092,18 @@
...
 
 	else if(href_list["deletemessageempty"])
 		if(!check_rights(R_ADMIN))
 			return
-		var/message_id = href_list["deletemessageempty"]
-		delete_message(message_id, browse = 1)
+		var/safety = alert("Delete message/note?",,"Yes","No");
+		if (safety == "Yes")
+			var/message_id = href_list["deletemessageempty"]
+			delete_message(message_id, browse = 1)
```
Comment (Cyberboss) [inline code/modules/admin/topic.dm]: TRUE

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by ExcessiveUseOfCobblestone** [comment]
It's in chat so you can just replace it.

I personally think having a confirmation for every deleted note is going to outweight the fringe case of someone accidentally deleting one.

**Comment by DaxDupont** [comment]
@ExcessiveUseOfCobblestone Just clicking yes takes a second. It's not like you're gonna be deleting much. It's better to have a safety against accidental deletion. Sure if you'd delete a lot of notes but I don't see that happening much.

**Comment by Iamgoofball** [comment]
@ExcessiveUseOfCobblestone which resets the note fade

**Comment by ExcessiveUseOfCobblestone** [comment]
@Iamgoofball If I ever did it I probably wouldn't readd it back unless it was something worth keeping byond "note fade" anyways.

I'm probably just being contrarian.

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**